### PR TITLE
update bundler to 2.2.22

### PR DESCRIPTION
**Pull request by pfeffer**
This PR updates Bundler to 2.2.22 and regenerates the Gemfile.lock to explicitly define the source of each gem.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -48,4 +48,4 @@ DEPENDENCIES
   fpm
 
 BUNDLED WITH
-   1.12.1
+   1.17.3
```
Comment (jchestershopify) [inline Gemfile.lock]: Doesn't seem to have upgraded to 2.2.22?

Comment (pfeffer) [reply Gemfile.lock]: oops, good catch! fixing

**Review by jchestershopify — APPROVED** [review]
LGTM.

**Review by doodzik — APPROVED** [review]

**Comment by doodzik** [comment]
This repo is a fork, did you mean to update the bundler version here?

The repo is also >2000 commits behind, maybe we need to archive it.

**Comment by rochlefebvre** [comment]
@pfeffer :high-five: 🫂

**Pull request closed without merging**